.markdownlintignore

```diff
@@ -15,11 +15,11 @@
 ########################
 
 # Logs
-logs
-*.log
-npm-debug.log*
-yarn-debug.log*
-yarn-error.log*
+**/logs
```

Comment:
Do you remember why these `**/`s were required in the first place? (I see you just reverted them for `prettier`)

I know the inconsistency was introduced in a previous PR but it'd be good if we didn't have to have actual different ignore patterns between the shared portion of the ignore configs..

Reply:
And have you noticed a difference between the behaviour of these ignore files vs `.gitignore`? I believe the behaviour of `.gitignore` is to match at any subdirectory if the pattern is not a path 

> If there is a separator at the beginning or middle (or both) of the pattern, then the pattern is relative to the directory level of the particular .gitignore file itself. **Otherwise the pattern may also match at any level below the .gitignore level.**

- https://git-scm.com/docs/gitignore

Reply:
For `.gitignore` this is not required, however, for `.dockerignore` it's required. As these files should be in sync I kept the syntax for `.gitignore`

Reply:
> As these files should be in sync I kept the syntax for .gitignore

Why did you [Revert .prettierignore](https://github.com/hashintel/hash/pull/2322/commits/fff01aca94d6eb4680f80303c8bf5c596148c893) then?

Reply:
Because we are generating `.eslintignore` based on that. Please see [this slack thread](https://hashintel.slack.com/archives/C02TWBTT3ED/p1680175990406509?thread_ts=1680167187.910809&cid=C02TWBTT3ED) for the context.

Reply:
Hm, it makes me quite uncomfortable to introduce even more inconsistency in this PR. Let's fix ASAP